src/modules/userDashboard/components/YourReferrals.tsx

```diff
@@ -0,0 +1,56 @@
+import { Paper, Stack } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import NotFound from '@/components/pages/NotFound';
+import useAuth from '@/modules/auth/hooks/useAuth';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import UserStepCard from '@/modules/userDashboard/components/UserStepCard';
+import {
+  GetUserCeAssignedStepsDocument,
+  GetUserCeAssignedStepsQuery,
+  GetUserCeAssignedStepsQueryVariables,
+  UserCeReferralStepFieldsFragment,
+} from '@/types/gqlTypes';
+
+const YourReferrals = () => {
+  const { user } = useAuth();
+  if (!user) return <NotFound />;
+
+  // TODO(#7507) - using this "table" to display cards results in some dom validation issues that cause console warning.
+  // When we refactor GenericTable to use hooks, we should consider supporting non-table layouts more gracefully.
+  return (
+    <>
+      <Paper>
+        <Stack gap={1} m={2}>
+          <Typography component='h2' variant='h4'>
+            Referrals
+          </Typography>
+          <Typography variant='caption'>
+            Referral steps assigned to you
+          </Typography>
+        </Stack>
```

Comment:
```suggestion
      <CommonCard title='Referrals' padContent={false} sx={{ pb: 1 }}>
```

- discussed at design standup: remove caption description
- I would suggest using CommonCard on the User Dashboard. I can't think of a reason we should use custom cards here. Using CommonCard will ensure that all of the cards on the dashboard have the same styles.